spec/requests/mentees_request_spec.rb

```diff
@@ -24,5 +24,15 @@
         expect(mentees["message"]).to eq("No mentee recorded")
       end
     end
+    context "add course" do
+      it "populate the mentee table with a course ID" do
```

Comment:
Table and ID are all implementation details. Can you think of a better test description that describes the behavior here?